rollout/analysis.go

```diff
@@ -460,6 +460,11 @@ func (c *rolloutContext) newAnalysisRunFromRollout(rolloutAnalysis *v1alpha1.Rol
 	for k, v := range rolloutAnalysis.AnalysisRunMetadata.Labels {
 		run.Labels[k] = v
 	}
+
+	for k, v := range c.rollout.Spec.Selector.MatchLabels {
```

Comment:
I think taking the labels on the rollout object vs the MatchLabels probably make more sense and follows what we do with ReplicaSets, with RS we take the pod template labels and put them on the RS as labels.

Reply:
Thanks @zachaller, I'll have a look at the RS code and follow the same practice.

Reply:
Sorry for the delay, as requested updated to use template labels.